### ISSUE TITLE
Didn't work for me, nothing rendered on the screen!

### ISSUE THREAD

**Issue by hack-and-backslash** [VincentCATILLON/react-native-confetti-cannon#1]
I was looking for such a thing since a week now and found this one. Unfortunately, it doesn't work for me. Neither does it throw any errors or log anything to the console. 

Using: 
    "react": "16.6.3",
    "react-native": "0.58.6",
    "react-native-confetti-cannon": "^1.0.2",

**Comment by VincentCATILLON** [comment]
Hey @hack-and-backslash sorry for the response delay.

I'll spend time to understand why it doesn't work on your own app.

Can you copy past the component using this lib ?

Mine is: 

```
import * as React from 'react';
import {View} from 'react-native';
import ConfettiCannon from 'react-native-confetti-cannon';

const MyScreen = () => (
  <View style={styles.container}>
    <ConfettiCannon count={200} origin={{x: -10, y: 0}} />
  </View>
);
```

**Comment by hack-and-backslash** [comment]
`import React, { Component } from 'react';
import { StyleSheet, View } from 'react-native';
import ConfettiCannon from 'react-native-confetti-cannon';

export default class App extends Component {
  render() {
    return (
      <View style={styles.container}>
        <ConfettiCannon count={200} origin={{ x: -10, y: 0 }} />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  }
});
`

**Comment by hack-and-backslash** [comment]
Here's what I tried: 

`import React, { Component } from 'react';
import { StyleSheet, View } from 'react-native';
import ConfettiCannon from 'react-native-confetti-cannon';

export default class App extends Component {
  render() {
    return (
      <View style={styles.container}>
        <ConfettiCannon count={200} origin={{ x: -10, y: 0 }} />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  }
});
`

**Comment by hack-and-backslash** [comment]
I am doing the same thing. Nothing gets rendered on screen for me.

Here's my class: 

export default class App extends Component {
  render() {
    return (
      <View style={styles.container}>
        <ConfettiCannon count={200} origin={{ x: -10, y: 0 }} />
      </View>
    );
  }
}

**Comment by VincentCATILLON** [comment]
Ok i'll see it ASAP.

Are you sure that you have a `View` wrapping the `ConfettiCannon` component with a height and width stuck to your window edges ?

**Comment by Tracklous** [comment]
In ios, it is working fine but in android its show nothing. I guess fragment is causing some problem to render any confetti.

**Comment by VincentCATILLON** [comment]
Hi @Tracklous & @hack-and-backslash i'm reproducing your issue locally.
With 0.59, there is no issue with iOS.

I'm looking to fix this ASAP and release a patch for this and back to you.

Don't know if it's related to this fix (in 0.59): https://github.com/facebook/react-native/pull/18872

**Comment by VincentCATILLON** [comment]
Ok guys, I identified the root cause.

In debug mode, it's working, but not without debug mode. It seems that it's related to https://github.com/facebook/react-native/issues/18426 

I'll find a workaround or change the code to fix this. Stay tuned

**Comment by VincentCATILLON** [comment]
@hack-and-backslash @Tracklous a new version was published (1.1.2). Please upgrade your lib and tell me if it's fixed on your project :) 

`yarn upgrade react-native-confetti-cannon@^1.1.2`

**Comment by Tracklous** [comment]
Hi! @VincentCATILLON ,

It is running on Android now, but after the little fix " Array(count).fill(count).map(...." I've to give the args to the fill() function so I just passed the count in it. But there are also some performance issues whenever I put the count to greater than 70-80 confetti's then it performs the animation in <60 FPS. 

Anyways thanx for the quick response.

**Comment by VincentCATILLON** [comment]
Yes, i think it depends on device CPU. We could use `useNativeDriver` for best performances but some transformations properties are not supported yet for this: `top`, `left` and `rotateZ`.

The solution could be to make it natively and expose it to JS through RN bridge.

**Comment by VincentCATILLON** [comment]
I'm closing the issue if it's right for you so :) Have a great day

**Comment by hack-and-backslash** [comment]
> 
> 
> @hack-and-backslash @Tracklous a new version was published ([1.1.2](https://www.npmjs.com/package/react-native-confetti-cannon)).
> Please upgrade your lib and tell me if it's fixed on your project :)
> 
> `yarn upgrade react-native-confetti-cannon@^1.1.2`

Eveything seems fine now. Thanks.

**Comment by jhoanborges** [comment]
Same issue. I'm using react-native-confetti-cannon@^1.1.2 and React native 0.59.10
